cmake/extensions.cmake

```diff
@@ -210,7 +210,14 @@ Please provide one of following: CONF_FILES")
     endif()
   endif()
 
-  zephyr_file(CONF_FILES ${NCS_FILE_CONF_FILES}/boards ${NCS_FILE_UNPARSED_ARGUMENTS})
+  set(additional_append)
+
+  if(DEFINED PREPROCESS_ARGS_BOARD)
+    parse_board_components(PREPROCESS_ARGS_BOARD board_name board_revision board_qualifiers)
+    set(additional_append BOARD ${board_name} BOARD_REVISION ${board_revision} BOARD_QUALIFIERS ${board_qualifiers})
+  endif()
```

Comment:
To be addressed before merging, see: https://github.com/gmarull/sdk-nrf/pull/13#discussion_r1570646695